console/console.go

```diff
@@ -276,20 +276,22 @@ func (c *Console) AutoCompleteInput(line string, pos int) (string, []string, str
 func (c *Console) Welcome() {
 	// Print some generic Geth metadata
 	fmt.Fprintf(c.printer, "Welcome to the Geth JavaScript console!\n\n")
-	c.jsre.Run(`
-		console.log("instance: " + web3.version.node);
-		console.log("coinbase: " + eth.coinbase);
-		console.log("at block: " + eth.blockNumber + " (" + new Date(1000 * eth.getBlock(eth.blockNumber).timestamp) + ")");
-		console.log(" datadir: " + admin.datadir);
-	`)
+	for _, command := range []string{
+		`console.log("instance: " + web3.version.node);`,
+		`console.log("coinbase: " + eth.coinbase);`,
+		`console.log("at block: " + eth.blockNumber + " (" + new Date(1000 * eth.getBlock(eth.blockNumber).timestamp) + ")");`,
+		`console.log("datadir:  " + admin.datadir);`,
+	} {
+		c.jsre.Run(command)
+	}
```

Comment:
I don't see a reason to run commands one-by-one, it's a programming language, lets make it work in js.